HealthcareAnalytics/HealthcareAnalytics/Views/Employees/Index.cshtml

```diff
@@ -61,3 +61,12 @@
 }
 
 </table>
+
+<ul class="pagination">
+    @for (int i = 1; i <= ViewBag.Total / ViewBag.ResultsPerPage; i++)
+    {
+        <li>
+            @Html.ActionLink(""+i.ToString("00"), "Index/" +i, i)
```

Comment:
What happens if we have 100,000 and per page is 10?  Will it show every single link?  This will look ugly.  Is it possible to only show ~5 with '...', 'last', 'first', etc?